collections/_posts/2022-11-02-NeighborhoodAttentionTransformer.md

```diff
@@ -0,0 +1,102 @@
+---
+layout: review
+title: "Neighborhood Attention Transformer"
+tags: deep-learning CNN transformer segmentation classification object-detection attention
+author: "Pierre Rougé"
+cite:
+    authors: "Ali Hassani, Steven Walton, Jiachen Li, Shen Li, Humphrey Shi"
+    title:   "Neighborhood Attention Transformer"
+pdf: "https://arxiv.org/abs/2204.07143v1"
```

Comment:
I typicall prefer not to refer to a specific version of an arxiv paper (the `v1` at the end here) so that URL always leads to the most recent version.